ecommerce/src/App.js

```diff
@@ -0,0 +1,38 @@
+import './neutron-ui.css';
+import './App.css';
+import {Routes,  Route} from 'react-router-dom';
+import Home from './Pages/Home';
+import Cart from './Pages/Cart';
+import Navbar from './Components/Navbar';
+import Products from './Pages/Products';
+import Login from './Pages/Login';
+import  { Toaster } from 'react-hot-toast';
```

Comment:
You can add index.js file in components folder to make import statements a bit cleaner